apps/web/src/components/professor-card.tsx

```diff
@@ -83,32 +87,63 @@ const SocialLinks = ({ professor }: ProfessorCardProps) => {
 
 export const ProfessorCard = ({ professor, ...props }: ProfessorCardProps) => {
   return (
-    <div {...props}>
-      <div className="border-blue-1000 bg-beige-300 flex h-full flex-col items-start gap-2.5 rounded-[1.25rem] border p-1">
-        <div className="flex w-[240px] grow flex-col">
-          <div className="border-blue-1000 flex flex-col rounded-t-2xl border bg-orange-500 px-5 py-2">
-            <span className="text-beige-300 break-words text-xl font-semibold">
+    <div
+      className="flex  flex-wrap lg:max-w-[280px] lg:max-h-[443px] p-2 md:p-7 gap-5 rounded-2xl border-t border-t-newGray-4 bg-newGray-6 shadow-course-navigation mt-8"
+      {...props}
+    >
+      <div className="rounded-[20px] flex max-md:flex-col max-md:max-w-[127px] max-md:gap-4 md:items-end gap-7 bg-white">
+        <div className="rounded-[20px] p-1 md:p-2 border-1 md:border-2 border-white">
+          <div className="rounded-[20px] py-4 flex flex-col items-center bg-gradient-to-b from-[#411800] to-[#FF5C00] p-2.5 md:w-[280px] relative overflow-hidden">
+            <span className="mb-8 w-full text-center text-base lg:title-large-sb-24px text-white lg:uppercase z-10 ">
```

Comment:
We should **absolute position** this to avoid layout shift when the teacher's name takes 2 lines (see layout for "Arman the Parman" case)

We can remove the margin bottom and put a margin top on the image instead to ensure that we won't have layout issue

We should create a class for mobile font properties (label-med-med-16px -> put it in global.css, inside ui package)